include/CoreText/CoreTextExport.h

```diff
@@ -1,6 +1,6 @@
 //******************************************************************************
 //
-// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
These are nice but again unrelated to the rest of these changes.  Pull these into a separate commit?